packages/binding-mqtt/test/mqtt-client-subscribe-test.ts

```diff
@@ -28,66 +27,71 @@ import { Servient, ExposedThing } from "@node-wot/core";
 import MqttBrokerServer from "../dist/mqtt-broker-server";
 import MqttClientFactory from "../dist/mqtt-client-factory";
 
-
-
-
-
 @suite("MQTT implementation")
 class MqttClientSubscribeTest {
-
-    @test.skip "should expose via broker"(done: Function) {
-
+    @test "should expose via broker"(done: Function) {
         try {
-
             let servient = new Servient();
+            var brokerUri;
+
+            if (process.env.matrix_os == "ubuntu-latest") {
+                brokerUri = "mqtt://localhost:1883";
+            } else {
+                brokerUri = "mqtt://test.mosquitto.org:1883"
+            }
```

Comment:
As I said in my other comment here it would be easier if you just do:
```js
brokerUri = process.env.broker || "mqtt://test.mosquitto.org:1883"
```